Makefile-az.mk

```diff
@@ -361,11 +385,15 @@ az-argvmlist: ## List current VMs owned by Karpenter
 	| jq '.data[] | .id'
 
 az-pprof-enable: ## Enable profiling
-	yq -i '.manifests.helm.releases[0].overrides.controller.env += [{"name":"ENABLE_PROFILING","value":"true"}]' skaffold.yaml
+	yq -i '.controller.env += [{"name":"ENABLE_PROFILING","value":"true"}]' karpenter-values.yaml
 
+# remove -source_path=/go/pkg/mod to focus source on provider
+SOURCE=-source_path=/go/pkg/mod -trim_path=github.com/Azure/karpenter-provider-azure
 az-pprof: ## Profile
-	kubectl port-forward service/karpenter -n "${KARPENTER_NAMESPACE}" 8000 &
-	sleep 2 && go tool pprof -http 0.0.0.0:9000 localhost:8000/debug/pprof/heap
+	kubectl port-forward service/karpenter -n "${KARPENTER_NAMESPACE}" 8080 &
+	sleep 2
```

Comment:
Not a blocker, but out of curiosity, did it make a difference to remove the && from the script?

Reply:
No functional difference, just clarity. `cmd1 && cmd2` is an idiomatic way to run cmd2 only if cmd1 succeeds, which is not the intent here, so I was abusing it a bit; cleaned up.